doc/html/python.html

```diff
@@ -3339,14 +3338,18 @@ <H3>
   (("cyrl","romn",(1405,-288,0),()),
    ("grek","romn",(1405,-288,0),()),
    ("latn","romn",(1405,-288,0),
-    ("dflt",-576,1913,
```

Comment:
What's happening here?

Reply:
Basically: 

1. I looked through `python.c` for code with suspicious string conversions and this function had some.
2. I looked for documentation on the interface so that I could test all the conversions and found this description.
3. I got the code so that I thought it should work but it couldn't parse the documented example. 
4. After trying and failing to get the interface to work many times I used the GUI to enter identical parameters into a font and then read it out using the same interface. It spit out something close to what I have documented. 
5. After looking back at the GUI and the working case I decided it made sense and the documentation didn't, so I edited the documentation to match the implementation.